.github/workflows/test.yml

```diff
@@ -15,7 +15,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [3.7, 3.8, 3.9]
+        python-version: ["3.8", "3.9", "3.10"]
```

Comment:
This will require changing the supported version of Python in `setup.py`

Reply:
Fixed.